pymongo/mongo_client.py

```diff
@@ -1793,7 +1822,7 @@ def get_database(self, name=None, codec_options=None, read_preference=None,
           Primary()
           >>> from pymongo import ReadPreference
           >>> db2 = client.get_database(
-          ...     'test', read_preference=ReadPreference.SECONDARY)
+          ...   'test', read_preference=ReadPreference.SECONDARY)
```

Comment:
Let's revert this line since this change is not needed.

Reply:
Done